amaznot/resources/views/auth/login.blade.php

```diff
@@ -0,0 +1,13 @@
+@extends('layouts.app')
+
+@section('content')
+    <div class = "flex justify-center">
+        <div class = "w-4/12 bg white p-6 rounded-lg">
+            <form action = "{{ route('login')}}" method = "post">
+
+            </form>
```

Comment:
Can you create a very simple form here and in the register page to ensure the authentication logic works as it should? (Unless you've verified it by some other means like Postman already)

Reply:
sure